zipline/src/commonMain/kotlin/app/cash/zipline/internal/bridge/outbound.kt

```diff
@@ -29,7 +29,7 @@ import okio.Buffer
  */
 @PublishedApi
 internal abstract class OutboundClientFactory<T : Any>(
-  internal val serializersModule: SerializersModule
```

Comment:
Were these made non-internal for dev / testing only?

Reply:
same